### ISSUE TITLE
Illustration for AnnData project download

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-docs#150]
We need to create a new illustration for the expected download structure when downloading a project as AnnData. 

Below is the expected structure copied from https://github.com/AlexsLemonade/ScPCA-admin/issues/595#issuecomment-1644330881

```
project_id
├── README.md
├── bulk_metadata.tsv
├── bulk_quant.tsv
├── sample_id
│   ├── SCPCL000000_filtered_rna.hdf5
│   ├── SCPCL000000_processed_rna.hdf5
│   ├── SCPCL000000_qc_report.html
│   └── SCPCL000000_unfiltered_rna.hdf5
└── single_cell_metadata.tsv
```

**Comment by dvenprasad** [comment]
How does this look?

<img width="385" alt="anndata-project-download-folder" src="https://github.com/AlexsLemonade/scpca-docs/assets/15315514/36d9c7fb-8b7d-4c78-afd0-6f373d043146">

**Comment by allyhawkins** [comment]
Looks good!

**Comment by allyhawkins** [comment]
Closed by #161
